### PR TITLE
functions for satellite pipeline

### PR DESCRIPTION
**Pull request by patchkez**

### CODE REVIEW

File: vars/clientUtils.groovy [hunk 1]
```diff
@@ -197,7 +198,7 @@ def setupIqePlugin(Map parameters = [:]){
         sh """
             echo "/iqe_venv exists, reusing it"
             source ${venvDir}/bin/activate
-            pip install -U pip setuptools setuptools_scm wheel iqe-integration-tests
+            pip install -U pip setuptools setuptools_scm wheel iqe-core
```
Comment (quarckster) [inline vars/clientUtils.groovy:201]:
```
pip install -U pip setuptools wheel
pip install setuptools_scm iqe-core
```

File: vars/clientUtils.groovy [hunk 2]
```diff
@@ -336,3 +343,18 @@ def runAnsible(String playbookFile, String playbookTags=null){
...
+
+def copySshKey(Map parameters = [:]){
+    def sshKey = parameters.get("sshKey", 'ssh_credentials_remediations_fifi')
+    def sshKeyName = parameters.get("sshKeyName", 'insights-qa.pem')
+    withCredentials([file(credentialsId: sshKey, variable: 'settings')]) {
+        sh "cp \$settings ~/.ssh/${sshKeyName}"
+    }
+}
+
+def downloadChromeDriver(){
```
Comment (quarckster) [inline vars/clientUtils.groovy:355]: Why do you download it every run? Would it be more efficient to have it in the image already?

**Review by quarckster — CHANGES_REQUESTED** [review]

File: vars/clientUtils.groovy (hunk 2 above)
Comment (patchkez) [reply vars/clientUtils.groovy]: I moved this to iqe-ansible repo which preconfigures/creates our Jslave Openstack images.

File: vars/clientUtils.groovy (hunk 1 above)
Comment (patchkez) [reply vars/clientUtils.groovy]: changed.

**Review by quarckster — APPROVED** [review]

**Pull request merged**